### DISCUSSION TITLE
My website has a 100GB database ( a lot of users ) and doesn't have any replication database yet. Should I use readyset?

### DISCUSSION THREAD

**Discussion by meotimdihia** [readysettech/readyset#1078] [Q&A]
Currently, I need to set up a read-only streaming replication database but I see Readyset was introduced.
Is Readyset production ready?

**Comment by MichalisDBA** [comment]
I want to ask if readyset for mysql is production ready too. We don't have complicated queries but they do a lot of full table scans sadly.
